### ISSUE TITLE
save failure

### ISSUE THREAD

**Issue by p7willm** [goatfungus/NMSSaveEditor#127]
I just downloaded 1.5.25 and get n error message box with the text " There was an error saving the file." with an icon with a quarter orange circle with a white block in the middle. The only button is Cancel. 

I got the error and exited the editor. I think I clicked yes to the save before leaving.

Next I restarted Windows, started the game, did a manual save, exited the game, started the editor, selected my save file, selected the only active save in the file, it matched the date/time of the save, went to the exosuit and deleted one item from inventory, went back, clicked Save changes, got the error. 

To get the text for the Save changes button I started the editor again, selected my file, selected slot 1, the only one in use, and immediately clicked Save changes and got the error. When I closed the app I did not get the save changes dialog, probably because I made no changes.

**Comment by goatfungus** [comment]
Please send your NMSSaveEditor.log file.

**Comment by Ba5ic** [comment]
Have the same problem ...I think since the latest patch

**Comment by p7willm** [comment]
Fixed it. 

Wouldn’t it be annoying if that was all I said. 

Went looking for the log file and couldn’t find it. Didn’t know where to look so I looked in documents, nms folder, etc. Realized I was running the editor out of the zip file I downloaded and it probably created the log in the same location it ran from and couldn’t create it in the zip file so moved the executable to the desktop and ran it from there. It worked fine. deleted an item from inventory, saved, ran the game, inventory showed up as saved. 

I had been using a slightly out of date version  run from a zip and it was working. The new one is better, with the log and configuration saved. I thought you had to select the edit file every time you ran the program. 

The config file is a nice touch I did not expect. I thought this just read the save and edited it, one step above going in and editing the  JSON file. There is unexpected depth to this program. 

Thanks for all the work. 

I have been doing a save, exiting the game, editing the file, starting the game again. Thinking about it I am going to try doing a save in the game, doing a reload in the editor, editing the file, saving, doing a load save in the game. Should be a lot easier and quicker.
